siliconcompiler/scheduler.py

```diff
@@ -19,10 +19,7 @@ def _deferstep(chip, step, index, status):
     scheduler_type = chip.get('option', 'scheduler', 'name', step=step, index=index)
 
     # Determine which cluster parititon to use. (Default value can be overridden on per-step basis)
-    if f'{step}_slurm_partition' in chip.status:
-        partition = chip.status[f'{step}_slurm_partition']
-    else:
-        partition = chip.status['slurm_partition']
+    partition = chip.get('option', 'scheduler', 'queue', step=step, index=index)
```

Comment:
documentation needs to be updated to reflect this is modified

Reply:
The Slurm documentation is still pretty barebones, and it currently only references the `('option', 'scheduler', 'name', 'slurm')` keypath.